docs/install/README.md

```diff
@@ -55,11 +57,14 @@ Please be aware that:
 
 For more details see the [built-in runtime configuration](../configuration/built-in-runtimes.md)
 
-In addition, the following resources will be created in the same namespace:
+In addition, the following resources will be created in the control plane namespace:
 
 - `model-serving-defaults` - ConfigMap holding default values tied to a release, should not be modified. Configuration can be overriden by creating a user ConfigMap, see [configuration](../configuration)
 - `tc-config` - ConfigMap used for some internal coordination
 - `storage-config` - Secret holding config for each of the storage backends from which models can be loaded - see [the example](../predictors/)
+- `model-serving-etcd` - Secret providing access to the Etcd cluster - see [instructions](../install/install-script.md#setup-the-etcd-connection-information)
```

Comment:
I thought "model-serving-etcd" is similar to "storage-config", up to the user to provide. The line above has "`storage-config` - Secret holding config for each of the storage backends from which models can be loaded - see [the example](../predictors/)". So the word "created" seems to include both by code and user.

Reply:
They are a bit different:
- The etcd secret must be created by the user as a prereq to installation and should be basically static and never (or very rarely) change. However _if_ the `--quickstart` install option is used it will run etcd and create the secret for you.
- The storage config secret does not need to be created by the user, an empty one will be created during installation if it doesn't exist. The user can update this (add/update/delete entries within it) and the system will pick up those changes dynamically. You just need to make sure you add a storage entry to the cert before it can be used by an InferenceService/Predictor. If the `--quickstart` option is used it will also set up minio for you and add an entry for it to the secret.

Reply:
Thanks for the clarification. If the session is about the resources created by code, not the user, the scenario for multi-namespace seems different. We actually create `model-serving-etcd` by copying one from the controller plane namespace and expect the user to create `storage-config`.